docs/adr/0002-persist-accounts-using-cs3-storage.md

```diff
@@ -0,0 +1,29 @@
+# 2. Persist accounts in a CS3 storage
+
+* Status: accepted <!-- optional -->
+* Deciders: @butonic, @felixboehm <!-- optional -->
+* Date: 2020-08-21 <!-- optional -->
+
+Technical Story: [File system based indexing](https://github.com/owncloud/ocis-accounts/pull/92) <!-- optional -->
+
+## Context and Problem Statement
+
+To set up HA or a geo replicated setup we need to persist accounts in a distributed way. Furthermore, the bleve index makes the accounts service stateful, which we wand to avoid for a scale out deployment.
```

Comment:
```suggestion
To set up HA or a geo replicated setup we need to persist accounts in a distributed way. Furthermore, the bleve index makes the accounts service stateful, which we want to avoid for a scale out deployment.
```